plugin-localization/src/main/java/com.mapbox.mapboxsdk.plugins.localization/LocalizationPlugin.java

```diff
@@ -24,15 +24,21 @@
 public final class LocalizationPlugin {
 
   private MapboxMap mapboxMap;
+  private String selectedBackupLanguage;
 
   /**
    * Create a {@link LocalizationPlugin}.
    *
-   * @param mapboxMap the MapboxMap to apply the localization plugin to
+   * @param mapboxMap      the MapboxMap to apply the localization plugin to
+   * @param backupLanguage the default language for the map to fall back to if the device language is not
+   *                       supported by Open Street Maps.
```

Comment:
Nit: OpenStreetMap supports every language that has an ISO 639 code. It’s the Mapbox Streets source that’s currently limited to [a handful of languages](https://www.mapbox.com/vector-tiles/mapbox-streets-v7/#overview).

Reply:
Fixed. Thanks